dffml/source/file.py

```diff
@@ -65,7 +65,7 @@ class FileSource(BaseSource):
         elif self.config.filename[::-1].startswith((".zip")[::-1]):
             opener = self.zip_opener_helper()
         else:
-            opener = open(self.config.filename, "r")
+            opener = open(self.config.filename, "rb")
```

Comment:
Circle back on this change, and make sure all the other sources (json, csv) are still working with this change. We probably also need to change the various compression openers from `rt` to `rb`